volume_project/extract/2022/rescu_sqls/rescu_centreline.sql

```diff
@@ -0,0 +1,161 @@
+-- Make a table that links RESCU detectors with the centreline from 2022-07-05
+CREATE TABLE scannon.rescu_cent_20220705 AS (
+
+-- get centreline segments
```

Comment:
Question! 
For rescu locations like this, do we want to join to both centreline or just one ?
![image](https://github.com/CityofToronto/bdit_volumes/assets/46324452/e2c2d0a4-efc6-4c0d-8b94-124fa43ce984)

Reply:
I don't know if the model would overweight counts for this sensor if its data was assigned to 2 centrelines